derp/derphttp/derphttp_client.go

```diff
@@ -1028,9 +1028,9 @@ var ErrClientClosed = errors.New("derphttp.Client closed")
 
 func parseMetaCert(certs []*x509.Certificate) (serverPub key.NodePublic, serverProtoVersion int) {
 	for _, cert := range certs {
-		if cn := cert.Subject.CommonName; strings.HasPrefix(cn, "derpkey") {
+		if cn, ok := strs.CutPrefix(cert.Subject.CommonName, "derpkey"); ok {
```

Comment:
not sure "cn" as a variable name is so accurate anymore, but also fine I suppose. Maybe "pubHex"?

Reply:
Went with `pubHex`